op-program/scripts/build-prestates.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
```

Comment:
I know this is my fault, but `readarray` doesn't exist in the Mac OS  version of bash which you can fix by installing bash via home-brew but this hardcoded path forces it back to using the Mac version.  Can we make this:

```suggestion
#!/usr/bin/env bash
```

and then it will use the homebrew version if installed (and I have finally learnt why this indirection is worthwhile...)